jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -434,8 +434,8 @@ def test_autoupdate_results(cubeviz_helper, spectrum1d_cube_largest):
 #    orig_med_flux = np.median(cubeviz_helper.get_data('extracted').flux)
 
     # replace Subset 1 with a larger subset, resulting fluxes should increase
-    cubeviz_helper.plugins['Subset Tools']._obj.combination_mode.selected = 'replace'
-    cubeviz_helper.plugins['Subset Tools']._obj.import_region(CircularROI(xc=5, yc=5, radius=3))
+    cubeviz_helper.plugins['Subsets'].combination_mode.selected = 'replace'
```

Comment:
We no longer need to use `combination_mode.selected` since `combination_mode` is now exposed as user api. You can instead use `combination_mode = 'replace'`.